config/locales/en.yml

```
@@ -99,6 +100,7 @@ en:
     loading: "Loading..."
     copy: "Copy to clipboard"
     copied: "Copied!"
+    unsupported: "Current view is not under a supported license. Provided as on-preview mode"
```

Maybe a little rewording:
```suggestion
    unsupported: "Beta features are not covered by the Enterprise Support license and are only provided as a technical preview."
```

---

Changed at https://github.com/SUSE/blue-horizon/pull/29/commits/224c68668e87aee8298eeb9630f20030daa1a416